src/main/java/com/google/ehub/data/ProfileDatastore.java

```diff
@@ -21,9 +21,6 @@
 
   public void addUserProfileToDatastore(String name, String email, String username, String bio) {
     Entity userEntity = new Entity(PROFILE_ITEM_KIND);
-    LoginServlet login_instance = new LoginServlet();
-    System.out.println("Login email: " + login_instance.getEmail());
-    System.out.println("email:" + email);
 
     userEntity.setProperty(NAME_PROPERTY_KEY, name);
     userEntity.setProperty(EMAIL_PROPERTY_KEY, email);
```

Comment:
I think you forgot to add the newly created entity into the datastore. Something like the following is missing:
```
datastore.put(userEntity);
```